.justfile

```diff
@@ -0,0 +1,178 @@
+#!/usr/bin/env just --justfile
+
+set dotenv-load := true
+
+repo := `git rev-parse --show-toplevel`
+profile := env_var_or_default('PROFILE', "dev")
+github-event-name := env_var_or_default('GITHUB_EVENT_NAME', "none")
+
+
+######################################################################
+## Helper to print a message when calling `just`
+######################################################################
+[private]
+@default:
+  echo "Usage: just <recipe>"
+  just list-repo-recipes
+  echo "For further information, run 'just --help'"
+
+# List recipes in this file and from the calling directory
+[no-cd]
+[private]
+@usage:
+  echo "Usage: just <recipe>"
+  just list-local-recipes
+  just list-repo-recipes
+  echo "For further information, run 'just --help'"
+
+[no-cd]
+[private]
+@list-local-recipes:
+  echo '\nAvailable recipes:'
+  just --list --unsorted --list-heading ''
+
+[private]
+@list-repo-recipes:
+  echo "\nRepository recipes:"
+  just --list --unsorted --list-heading ''
+
+
+######################################################################
+## Helper to run a global command
+######################################################################
+
+# Runs yarn in the repository root
```

Comment:
Just so I understand, what about this forces it to be in the repository root?

Reply:
It does not have `[no-cd]`, so it will be run from the `.justfile` directory.

Reply:
I guess I'm unclear on how this differs from other commands within this `.justfile` and why this one is singled out as being the repository root

Reply:
Other commands, such as `lint` or `format` has the `[no-cd]` attribute, which implies when invoking these jobs from a subdirectory it will not change the directory to the root. This implies, that when calling `just format` inside of `apps/hash-graph` it will not run `format` in the repository root but in `apps/hash-graph`. However, we want `yarn` to be executed in the workspace root, so it's possible to call `just yarn` from anywhere.